lms/djangoapps/ccx/views.py

```diff
@@ -94,8 +95,8 @@ def wrapper(request, course_id):
 
         # if there is a ccx, we must validate that it is the ccx for this coach
         if ccx is not None:
-            coach_ccx = get_ccx_for_coach(course, request.user)
-            if coach_ccx is None or coach_ccx.id != ccx.id:
+            coach_ccx = get_ccx_by_ccx_id(course, request.user, ccx.id)
+            if coach_ccx is None:
                 return HttpResponseForbidden(
```

Comment:
By the way I was trying to replace code via  ``from django.core.exceptions import PermissionDenied`` but i fount that we can not add custom messages to this exception. To customize exception we need to implement multiple 403.html pages. which is a pain to manage.

https://docs.djangoproject.com/en/dev/ref/views/#the-403-http-forbidden-view
 
Any suggestion?